opennms-webapp/src/main/webapp/about/index.jsp

```diff
@@ -135,6 +137,21 @@
     <th>Time-Series Strategy:</th>
     <td><%=TimeSeries.getTimeseriesStrategy().getDescr()%></td>
   </tr>
+<%
+    String globalMBStrategy, sinkMBStrategy, rpcMBStrategy;
+    globalMBStrategy = org.opennms.core.rpc.common.RpcStrategy.getRpcStrategy().getName();
+    if (globalMBStrategy != null) {
```

Comment:
Didn't get why you still need `globalMBStrategy` here.

Reply:
Think you are trying to use `globalMBStrategy` in order to get both strategy names to be `jms`. Instead I think we can change `CAMEL_SINK_STRATEGY_NAME` to `jms`

Reply:
Guess I misunderstood earlier in our conversation. In this case, the scriptlet is totally unnecessary. Will change accordingly.